src/ui/pages/Overview/Overview.tsx

```diff
@@ -314,7 +314,10 @@ function OverviewComponent() {
               icon={
                 <WalletSourceIcon
                   address={singleAddress}
+                  groupId={null}
```

Comment:
Not sure is it a design requirement, but imho it would be nice to see wallet type here too.

Reply:
But you do! :) we just don't know which group the "current address" belongs to, so we would just get the first one and display the icon for it

Reply:
Oh, sorry, forgot to delete comment after checking old version